### PR TITLE
test: fix run-test and setup

### LINKED ISSUE #1000: Errors when running run_tests.sh
**Describe the issue**
The `run_tests.sh` fails with this error:
```

    def test_document_person_resolve_exception(es_clear, db, document_data_ref):
        """Test document person resolve."""
        with pytest.raises(Exception):
            Document.create(
                data=document_data_ref,
                delete_pid=False,
                dbcommit=True,
>               reindex=True
            )
E           Failed: DID NOT RAISE <class 'Exception'>

tests/ui/documents/test_documents_api.py:147: Failed
```

### PR DESCRIPTION
**Pull request by rerowep**
* Fixes problem with ui/documents/test_documents_api.py noz raising
  exceptions. closes #1000
* Changes setup back to not run the ebooks harvesting for normal setup.
* Corrects unused imports in scripts/__init__.py

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- https://github.com/rero/rero-ils/issues/1000

## How to test?

- run run-test
- run setup without connection to ebooks

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: scripts/__init__.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 """Poetry script utils."""
 
 import os
-import subprocess
+# import subprocess
```
Comment (BadrAly) [inline scripts/__init__.py:21]: delete this line

File: tests/ui/documents/test_documents_api.py [hunk 2]
```diff
@@ -139,6 +139,10 @@ def test_document_can_delete_with_loans(
 
 def test_document_person_resolve_exception(es_clear, db, document_data_ref):
     """Test document person resolve."""
+    document_data_ref['authors'] = [{
```
Comment (BadrAly) [inline tests/ui/documents/test_documents_api.py:142]: better to  add this in the file `tests/data/data.json` document= `doc2`?

File: scripts/__init__.py (hunk 1 above)
Comment (rerowep) [reply scripts/__init__.py:21]: Then I have to delete all the comments!

File: tests/ui/documents/test_documents_api.py (hunk 2 above)
Comment (rerowep) [reply tests/ui/documents/test_documents_api.py:142]: No, because I like to trigger only on this test the exception.

**Comment by blankoworld** [comment]
I got this error on bootstrap:

```
File "/home/od/rero/ils/rero_ils/schedulers.py", line 24, in <module>
    import jsonpickle
ModuleNotFoundError: No module named 'jsonpickle'
```

File: scripts/setup [hunk 3]
```diff
@@ -370,8 +370,10 @@ then
     eval ${PREFIX} invenio oaiharvester harvest -n ebooks -q -k
```
Comment (blankoworld) [inline scripts/setup:370]: Should not be mandatory if we make a deployment, but that ebooks are not needed.

File: scripts/setup [hunk 4]
```diff
@@ -370,8 +370,10 @@ then
...
 else
     eval ${PREFIX} invenio scheduler enable_tasks -n bulk-indexer -n notification-creation -n accounts -n clear_and_renew_subscriptions -v
-    info_msg "Start OAI harvesting 100"
-    eval ${PREFIX} invenio oaiharvester harvest -n ebooks -a max=100 -q
+    info_msg "For ebooks harvesting run:"
+    msg "\tinvenio oaiharvester harvest -n ebooks -a max=100 -q"
+    # info_msg "Start OAI harvesting 100"
+    # eval ${PREFIX} invenio oaiharvester harvest -n ebooks -a max=100 -q
```
Comment (blankoworld) [inline scripts/setup:376]: I suggest:

1. do not make ebook harvesting by default
2. add a parameter that activate the ebook harvesting. In deployment or not.

**Pull request merged**